tests/phpunit/Civi/Payment/PropertyBagTest.php

```diff
@@ -13,7 +13,10 @@ class PropertyBagTest extends \PHPUnit\Framework\TestCase implements HeadlessInt
    * @return \Civi\Test\CiviEnvBuilder
    */
   public function setUpHeadless() {
-    return \Civi\Test::headless()->apply();
+    static $reset = FALSE;
```

Comment:
@artfulrobot why is this needed ?

Reply:
@artfulrobot how does that fit with setupBeforeClass ? That seems to be meant to do what you want.

https://phpunit.de/manual/6.5/en/fixtures.html#fixtures.more-setup-than-teardown